funfact/lang/_terminal.py

```diff
@@ -152,7 +152,7 @@ class TensorSymbol(Symbol):
         _anon_registry = {}
         _anon_registry_lock = multiprocessing.Lock()
 
-    def __init__(self, *size, symbol=None, initializer=None):
+    def __init__(self, *size, symbol=None, initializer=None, optimizable=None):
```

Comment:
Since `optimizable` is a boolean, the default value should be either `True` or `False`.